scheduler/job.c

```diff
@@ -596,9 +596,9 @@ cupsdContinueJob(cupsd_job_t *job)	/* I - Job */
       }
 
       if (dst)
-        cupsdLogJob(job, CUPSD_LOG_DEBUG, "Retrying job as \"%s\".", strchr(dst->type, '/') + 1);
+        cupsdLogJob(job, CUPSD_LOG_DEBUG, "Forcing job as \"%s\".", strchr(dst->type, '/') + 1);
```

Comment:
Here it would be nice to differentiate between "print-as-raster" and "cups-retry-as-raster" - maybe -1 is retry and 1 is print as raster? Then the log message can be suitably modified.